### DISCUSSION TITLE
Generated report is missing HTML file

### DISCUSSION THREAD

**Discussion by petrac-daniel** [serenity-bdd/serenity-core#3366] [Q&A]
Hi

When I generate a report with serenity, I cannot open links that describe the call flow of one scenario.

For example if 1 scenario was executed and when I click on it, it tries to open the page 43290caf3b3e019161305bafe1a5bfc9f64b7e17333e35e02fdef480f252bde9.html. But the HTML file was not generated. I can find the .json file with that name, but not the HTML for it.
The old version of serenity generated both files, the .html file and .json so the preview of the scenario execution in the browser worked.

Why could this be happening? I am looking into the source, but I only found JSONTestOutcomeReporter, but this only generates and writes out the JSON file.

I am trying to migrate from 2.0.50 to serenity version 4.0.3

Thank you.

**Comment by wakaleo** [comment]
In Serenity 4.x you need to  run mvn serenity:aggregate to generate the HTML reports

**Reply by petrac-daniel** [reply]
In my case the HtmlAggregateStoryReporter was instantiated directly in java and like this the report was generated.

I figured out that this line was missing:
    reporter.setGenerateTestOutcomeReports();
    
Now it works, thank you so much!
